Bio/PDB/FragmentMapper.py

```diff
@@ -166,16 +156,14 @@ def __sub__(self, other):
         return sup.get_rms()
 
     def __repr__(self):
-        """
-        Returns <Fragment length=L id=ID> where L=length of fragment
+        """Returns <Fragment length=L id=ID> where L=length of fragment
```

Comment:
Rather insert a simple one line summary and keep the existing text as the main body of the docstring. How about ``String representing the fragment object.`` here?